cmd/cmd.go

```diff
@@ -50,3 +59,14 @@ func setGUID() error {
 	}
 	return nil
 }
+
+func setOutputType(f string) (string, error) {
+	if f == "" {
```

Comment:
```
if f == "" {
  return outputTypeText, nil
}

if strings.Compare(f, outputTypeJSON) != 0 && strings.Compare(f, outputTypeTEXT) != 0 && strings.Compare(f, outputTypeXML) != 0 {
  return "", ErrInvalidMessage
}

return f, nil
```

also, why `strings.Compare` and not just `f != outputTypeJSON`?

Reply:
Normal comparison `==` wasn't giving me the correct result!

Seems to now o.O